org.vranki.spectral.yaml

```diff
@@ -0,0 +1,23 @@
+id: org.vranki.spectral
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: spectral
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-download
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+modules:
+  - name: spectral
+    buildsystem: qmake
+    sources:
+      - type: git
+        url: https://gitlab.com/vranki/spectral.git
+        branch: develop
```

Comment:
Please make a release and update appdata accordingly. Other than that, LGTM.

Reply:
Added tag & commit to yaml.